.github/workflows/integrationTests-2.x.yaml

```diff
@@ -0,0 +1,132 @@
+name: Scheduled Integration Tests for 2.x
+
+on:
+  # push to 2.x branch is handled by default integrationTests.yaml
+  workflow_dispatch:
+  schedule:
+    - cron: '05 5 * * *' # 05:00 UTC every day
+
+
+jobs:
+  integrationTests:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        it:
+          - bigquery
+          - cloudsql
+          - config
+          - datastore
+          - firestore
+          - kms
+          - logging
+          - metrics
+          - multisample
+          - kotlin
+          - pubsub
+          - pubsub-bus
+          - pubsub-docs
+          - pubsub-emulator
+          - pubsub-integration
+          - secretmanager
+          - spanner
+          - storage
+          - trace
+          - vision
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d' --utc)"
+      - uses: actions/checkout@v2
+        with:
+          ref: '2.x'
```

Comment:
Not sure how tolerant GHA is on config. According to this guide I found, branch name goes without quotes?
https://github.com/actions/checkout#Checkout-a-different-branch